include/cantera/zeroD/flowControllers.h

```diff
@@ -15,6 +15,14 @@ namespace Cantera
 /**
  * A class for mass flow controllers. The mass flow rate is constant or
  * specified as a function of time.
+ *
+ * The device coefficient *c* is specifies the mass flow coefficient with units of kg/s.
+ * The mass flow rate is computed as:
+ * @f[\dot{m} = m g(t) @f]
+ * where *g* is a function of time that is set by `setTimeFunction`.
+ * If no function is specified, the mass flow rate defaults to:
+ * @f[\dot{m} = m @f]
+ *
```

Comment:
```suggestion
 *
 * The device coefficient *c* specifies the mass flow coefficient with units of kg/s.
 * The mass flow rate is computed as:
 * @f[\dot{m} = c g(t) @f]
 * where *g* is a function of time that is set by `setTimeFunction`.
 * If no function is specified, the mass flow rate defaults to:
 * @f[\dot{m} = c @f]
 *
```
Double check me on these latter two. Also, it might help to link to the `deviceCoefficient` / `setDeviceCoefficient` methods, as it's not otherwise obvious where this comes from.

Reply:
Confirming copy-pasta.